radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/epoch/EpochManager.java

```diff
@@ -371,6 +353,11 @@ public void processConsensusEvent(ConsensusEvent consensusEvent) {
     this.processConsensusEventInternal(consensusEvent);
```

Comment:
What are your thoughts on maybe calling this `processConsensusEventForCurrentEpoch`?